src/openrct2-ui/interface/ViewportInteraction.cpp

```diff
@@ -433,7 +434,7 @@ InteractionInfo ViewportInteractionGetItemRight(const ScreenCoordsXY& screenCoor
             auto banner = tileElement->AsBanner()->GetBanner();
             if (banner != nullptr)
             {
-                auto* bannerEntry = GetBannerEntry(banner->type);
+                auto* bannerEntry = OpenRCT2::ObjectManager::GetObjectEntry<BannerSceneryEntry>(banner->type);
```

Comment:
Why not just include a "using namespace OpenRCT2;"?

Reply:
It's considerably easier to do a find and replace when doing these changes if i include the namespace. We want to put all of openrct2 in the openrct2 namespace eventually but there are quite a few places where it can't just be added. so it's much easier for me to just have the namespace specified in all cases not just the problem ones and have a using in the none problem ones.

Reply:
Is there an issue or epic open for the namespace standardization?

Reply:
No, should probably create one. I tried to stick everything in OpenRCT2 namespace at the end of the snake_case refactor week but it proved much too hard due to name collisions.